drivers/i2c/i2c_ll_stm32.c

```diff
@@ -241,9 +241,9 @@ static int i2c_stm32_init(const struct device *dev)
 	}
 #endif /* CONFIG_SOC_SERIES_STM32F3X) || CONFIG_SOC_SERIES_STM32F0X */
 
-#if defined(CONFIG_SOC_STM32F103X8) || defined(CONFIG_SOC_STM32F103XB)
+#if defined(CONFIG_SOC_STM32F103X8) || defined(CONFIG_SOC_STM32F103XB) || defined(CONFIG_SOC_STM32F103XE)
```

Comment:
Please reduce line length :
```suggestion
#if defined(CONFIG_SOC_STM32F103X8) || \
     defined(CONFIG_SOC_STM32F103XB) || \
     defined(CONFIG_SOC_STM32F103XE)
```